expose.go

```diff
@@ -10,6 +14,21 @@ type ExposeDefinition struct {
 	TargetPort generictypes.DockerPort `json:"target_port,omitempty" description:"Port of the given component that implements the stable API."`
 }
 
+func (ed *ExposeDefinition) String() string {
+	m := map[string]string{
+		"port":        ed.Port.String(),
+		"component":   ed.Component.String(),
+		"target_port": ed.TargetPort.String(),
+	}
+
+	raw, err := json.Marshal(m)
```

Comment:
It does. Looks like the JSON implementation takes care about that: http://play.golang.org/p/U8nDgdga2X